apps/idos-enclave/src/pages/App.tsx

```diff
@@ -139,6 +142,7 @@ export function App({ store, enclave }: AppProps) {
     onError,
     onSuccess,
     mode,
+    currentUserPublicKey,
```

Comment:
We're passing this to the `Passkey` component, but we're not using it there. That smells fishy.

Reply:
I'd argue that's us circumventing the type system. I'd rather have a `MethodProps<{ password: string; duration: number }> & {expectedUserEncryptionPublicKey: string}` in the `Password` component.

Opposition?